packages/flutter_test/lib/src/binding.dart

```diff
@@ -611,6 +642,26 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
   late StackTraceDemangler _oldStackTraceDemangler;
   FlutterErrorDetails? _pendingExceptionDetails;
 
+  _MockMessageHandler? _announcementHandler;
+  List<CapturedAccessibilityAnnouncement>? _announcements;
+
+  /// Returns a list of all the accessibility announcements made by the Flutter
+  /// framework since the last time this function was called.
+  ///
+  /// It's safe to call this when there hasn't been any announcements; it will return
+  /// null in that case.
```

Comment:
Do we need `null` as a possible return value? An empty list should already be able to express the same thing. There would be less to document, and it would make the user code simpler since it won't need to handle nulls.

Reply:
Good point. I'll change it to return an empty list instead of null.